src/main/java/yoda/command/Command.java

```diff
@@ -4,23 +4,46 @@
 import yoda.ui.Ui;
 import yoda.storage.Storage;
 
+/**
+ * Command class to handle user inputs.
+ */
 public abstract class Command {
+    /** Details of user input */
     protected String[] details;
-    protected Input taskType;
+    /** Type of command referred to by user input */
+    protected CommandType taskType;
+    /** Shows if the user is done using the Yoda chatbot */
     protected boolean isExit;
 
+    /**
+     * Creates a command object.
+     */
     public Command() {
         this.isExit = false;
     }
 
+    /**
+     * Creates a command object.
+     * @param details Details required to handle user input.
+     */
     public Command(String[] details) {
         this.details = details;
         this.isExit = false;
     }
 
+    /**
+     * Checks if the user is done using the Yoda chatbot.
+     * @return Whether the user is done using the Yoda chatbot.
+     */
     public boolean isExit() {
```

Comment:
This method seems like a getter used to fetch the value of isExit variable. Perhaps you could consider changing this to getIsExit() so as to avoid confusions between the class variable and class method.